IdentityCore/src/oauth2/aad_v1/MSIDAADV1Oauth2Factory.m

```diff
@@ -106,33 +106,18 @@ - (BOOL)verifyResponse:(MSIDAADV1TokenResponse *)response
 
     if (!result)
     {
-        if (response.error)
+        // In case of not overriden error code, change it to default error code for v1.
+        if (*error &&
```

Comment:
small suggestion for making it a bit nicer: pass your own error to super verifyresponse and operate on it. Then, in the end assign final error to developer assigned error.